corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -1337,19 +1337,18 @@ def _get_default_form_report_columns(self):
             data_source_field=prop.to_report_column_option().get_indicator(COUNT_PER_CHOICE)['column_id'],
             calculation=COUNT_PER_CHOICE
         ))
-        for prop in self.data_source_properties.values():
-            questions_found = 0
-            if prop.get_type() == PROPERTY_TYPE_QUESTION:
-                questions_found += 1
-                cols.append(ColumnViewModel(
-                    display_text=prop.get_text(),
-                    exists_in_current_version=True,
-                    property=prop.get_id(),
-                    data_source_field=prop.get_id(),
-                    calculation=COUNT_PER_CHOICE,
-                ))
-                if questions_found == 4:
```

Comment:
Curious, what was this about?

Reply:
it was trying to do the question limit and failing